libs/application/core/src/types/ApplicationTemplate.ts

```diff
@@ -26,6 +26,7 @@ export interface ApplicationTemplate<
   readonly name: StaticText
   readonly institution?: StaticText
   readonly translationNamespaces?: string[]
+  readonly allowedDelegations?: string[]
```

Comment:
It might be a good idea to have an enum here, and for V1 we might want it to not support Custom delegations...